tests/test_bibtex_import.py

```diff
@@ -1,5 +1,39 @@
+from pathlib import Path
+
+import bibtexparser
+from bibtexparser.bparser import BibTexParser
+
 from academic import cli
+from academic import import_bibtex
+from academic.editFM import EditableFM
+
+
+bibtex_dir = Path(__file__).parent / 'data'
 
 
 def test_bibtex_import():
     cli.parse_args(["import", "--dry-run", "--bibtex", "tests/data/article.bib"])
+
+
+def _process_bibtex(file, expected_count=1):
```

Comment:
Please add some comments to this file to help with review and for future contributors to understand. Thanks.

Reply:
Are these comments sufficient?

Reply:
Thanks. Do we still need `from enum import Enum` in `import_bibtex`?

Reply:
Sorry no that somehow got reintroduced when reading. Let me push a fixed version

Reply:
Please run the lint, format, and tests when submitting PRs.

`make lint` is also failing due to `./academic/editFM.py:27:55: F841 local variable 'f' is assigned to but never used`. Please can you fix?

Thanks.

Reply:
Appears that was caused by #75. Sorry! Fix is in #79